viewer/src/assets/scss/theme.scss

```
@@ -61,11 +63,13 @@ $panel-left-txtcolor: $primary;
 $command-buttons-bgcolor: $palette-700;
 $content-bgcolor: $palette-900;
 $scrollbar-color: $palette-300;
+$scrollbar_width: 10px;
```

```suggestion
$scrollbar-width: 10px;
```

Kebab-case for consistency.